Gems/Atom/Feature/Common/Assets/Materials/Types/StandardMultilayerPBR_Displacement.lua

```diff
@@ -138,7 +138,8 @@ function CalcOverallHeightRange(context)
         if(enableLayer3) then GetMergedHeightRange(heightMinMax, offsetLayer3, factorLayer3) end
 
     else
-        heightMinMax = {0,0}
+        heightMinMax[0] = 0
+        heightMinMax[1] = 0
```

Comment:
I see. Lua indexing starts at 1, not 0 (this is very easy to forget!). So heightMinMax = {0,0} was setting indices 1 and 2 to "0", not 0 and 1, so heightMinMax[0] was returning nil.

If you don't mind, let's also change line 91 to be "local heightMinMax = {}" instead of "local heightMinMax = {nil, nil}" (same result, just cleaner initialization)

Reply:
Done.